development-docs/DEV_GUIDE.md

```diff
@@ -59,13 +59,18 @@ To build Strimzi from a source the operator and Kafka code needs to be compiled
 
    This updates `060-Deployment-strimzi-cluster-operator.yaml`, replacing all the image references (in `image` and `value` properties) with ones with the same name but with the repository changed.
 
-5. Then you can deploy the Cluster Operator by running: 
 
-        kubectl create -f install/cluster-operator
+5. The installation files assume you're installing into the namespace `myproject`.  If you wish to use a different one, you'll need to replace it in the installation files.
 
-6. Finally, you can deploy the cluster custom resource running:
+       sed -Ei "s/myproject/<desired_namespace>/g" install/cluster-operator/*.yaml
```

Comment:
* I think it should use `packaging/install/cluster-operator/*.yaml`
* Is this a Linux or MacOS variant? It might be good to cover both.

Reply:
Should I update step 4 (the image locations) to use `packaging/install ` over `install/` also? I can add the OS X sed variant for that step too.

Reply:
Yeah, looks like this is wrong also in steps 4, 6 ad 7. Please update it everywhere.

Reply:
Will do :)

Reply:
Have updated through out the doc. There's a bit of duplication between the Quick Start and the Building Strimzi, I'll push up another PR (branched from this one) with a proposed restructure.